frontend/projects/ui/src/app/modals/markdown/markdown.page.ts

```diff
@@ -24,7 +25,11 @@ export class MarkdownPage {
   async ngOnInit() {
     try {
       if (!this.content) {
-        this.content = await this.embassyApi.getStatic(this.contentUrl)
+        if (this.contentUrl) {
+          this.content = await this.embassyApi.getStatic(this.contentUrl)
```

Comment:
should this be using `marketplaceProxy` not `getStatic`?

Reply:
it should still be calling get static

Reply:
ok - so, for my understanding, both `getStatic` and `marketplaceProxy` utilize the rpc interface?

Reply:
getStatic is http